SMLHelper/Options/Utility/Validator.cs

```diff
@@ -24,24 +24,12 @@ internal static bool ValidateChoiceOrDropdownOption(string id, string label, str
             return true;
         }
 
-        internal static bool ValidateID(string id, out string result)
-        {
-            result = ValidateID(id);
-            if (result == null) return true;
-            return false;
-        }
-        internal static bool ValidateLabel(string id, out string result)
-        {
-            result = ValidateLabel(id);
-            if (result == null) return true;
-            return false;
-        }
-        internal static bool ValidateArray(string[] array, int index, out string result)
-        {
-            result = ValidateArray(array, index);
-            if (result == null) return true;
-            return false;
-        }
+        internal static bool ValidateID(string id, out string result) => (result = ValidateID(id)) == null;
+
+        internal static bool ValidateLabel(string id, out string result) => (result = ValidateLabel(id)) == null;
+
+        internal static bool ValidateArray(string[] array, int index, out string result) => (result = ValidateArray(array, index)) == null;
+
```

Comment:
Whatever auto formatting happened here made this even less readable than how AlexejheroYTB would normally have done it.

Please using code blocks for these instead of lambdas. Example
```CS
 internal static bool ValidateID(string id, out string result)
{
    result = ValidateID(id);
    return result == null;
}
```

Reply:
I looked at it after some time, and idk, maybe you right :)